migrate/sqlMigrator/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  datastore_emulator:
```

Comment:
why was this added?

Reply:
Added for the datastore viewer as it's sort of required for testing. I can remove it from the project if you think it's not necessary, and maybe leave a comment about the datastore viewer somewhere.